src/cdk/stepper/step.ts

```diff
@@ -0,0 +1,56 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {
+    Component, ContentChild, Directive, Input, OnInit, TemplateRef, ViewChild,
+    ViewContainerRef
+} from '@angular/core';
+import {CdkStepLabel} from './step-label';
+
+@Component({
+    selector: '[cdk-step]',
+    templateUrl: 'step.html',
+})
+export class CdkStep {
+    @ContentChild(CdkStepLabel) stepLabel: CdkStepLabel;
+    @ViewChild(TemplateRef) content: TemplateRef<any>;
+
+    /** Label of the step. */
+    @Input()
+    label: string;
+
+    /** Whether the step is optional or not. */
+    @Input() optional: boolean = false;
+
+    /** Whether the step is editable or not. */
+    @Input() editable: boolean = true;
+
+    /** Whether the step is the last one in the list. */
+    isLast: boolean = false;
```

Comment:
is this something we want to be part of the public API? (I asusme probably not, but we can't make it private since its referenced in a template file, so instead check to `_isLast`)